src/base/static/components/atoms/buttons.js

```diff
@@ -200,13 +200,13 @@ const Button = styled(props => {
   }
 
   if (props.color === "primary") {
+    // NOTE: hard-coded colors here are temporary, for a description of planned
+    // themeing changes see: https://github.com/jalMogo/mgmt/issues/329
     styles.backgroundColor = props.theme.brand.primary;
-    styles.color = props.theme.text.secondary;
+    styles.color = "#fff";
     styles["&:hover"].textDecoration = "none";
     styles["&:hover"].backgroundColor = props.theme.brand.primary;
-    styles["&:hover"].color = getReadableColor(
-      styles["&:hover"].backgroundColor,
-    );
```

Comment:
@jalMogo -- I went ahead and switched out this use of `getReadableColor` for a hard-coded value. This was creating some very harsh color clashes on almost all of our flavors' `AddPlaceButton`s when hovered. I think we're safe hard-coding `#fff` here for the text color since all flavors are designed to render that text in white, at least for the time being.

Longer-term I hope to resolve this along the lines of what's discussed in https://github.com/jalMogo/mgmt/issues/329.

Hopefully this doesn't throw off something else, like on the `mississippi` flavor or elsewhere. I took a pass through `mississippi`, but lmk if anything looks off and I'll try to resolve it.